app/controllers/api/v1/short_form_controller.rb

```diff
@@ -199,7 +199,7 @@ def submit_lease
       # from the supp app page, that will have created a blank lease before the
       # entire supp app form was saved.
       existing_lease = soql_lease_service.application_lease(application_api_params[:id])
-      if existing_lease
+      if existing_lease.present?
```

Comment:
soql_lease_service returns {} if the lease doesn't exist.

Reply:
easy enough!